src/peer_connection/certificate.rs

```diff
@@ -71,7 +72,14 @@ impl RTCCertificate {
         };
         params.key_pair = Some(key_pair);
 
-        let expires = params.not_after.into();
+        let expires = if cfg!(armv7) {
```

Comment:
are you sure cfg!(armv7) is valid target_arch?

from https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch, only the following arch are supported:

"x86"
"x86_64"
"mips"
"powerpc"
"powerpc64"
"arm"
"aarch64"

Reply:
oh, just found this link: https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-2-with-host-tools

Reply:
Before accept this PR, I just want to confirm it: have you build and test this change in armv7 system?

Reply:
Thanks for verifying.